QuickScannerApp/app/src/main/java/com/example/quickscanner/ui/scanner/ScannerFragment.java

```diff
@@ -213,28 +198,28 @@ private boolean isValidQRCode(String qrCodeValue) {
     }
 
     // Method to perform check-in for the current user
-//    private void checkInUser(String eventId) {
-//        // Get the current user's UID
-//        String userId = FirebaseAuth.getInstance().getCurrentUser().getUid();
-//
-//        // Call the checkIn method from FirebaseController to add the user to the check-ins array for the event
-//        //FirebaseController firebaseController = new FirebaseController();
-//        firebaseController.checkIn(eventId, userId);
-//
-//        // Inform the user that they have checked in successfully
-//        Toast.makeText(getContext(), "Checked in successfully!", Toast.LENGTH_SHORT).show();
-//
-//        // Launch MyEventsActivity and pass the event ID
-//        // TODO: Replace with the actual activity to launch
-//        /*
-//        Intent intent = new Intent(getContext(), MyEventsActivity.class);
-//        intent.putExtra("eventID", eventId);
-//        startActivity(intent);
-//         */
-//
-//        // Launch the event details activity
-//        launchEventDetails(getContext(), eventId);
-//    }
+    private void checkInUser(String eventId) {
+        // Get the current user's UID
+        String userId = FirebaseAuth.getInstance().getCurrentUser().getUid();
+
+        // Call the checkIn method from FirebaseController to add the user to the check-ins array for the event
+        //FirebaseController firebaseController = new FirebaseController();
+        firebaseController.checkIn(eventId, userId);
```

Comment:
Can you just use the firebase attendance controller to check in instead of the deprecated general controller?

Reply:
yes, i9 would prefer that as well. i just removed all references to it in my branch and was planning to push that to main

Reply:
If you reviewed the code, you can see I do not use that function. It was just there because it didn't affect the code (and just in case). I just removed it for the clear code readability.